packages/api/package.json

```diff
@@ -46,15 +47,16 @@
     "@nftstorage/ipfs-cluster": "^3.3.1",
     "@web3-storage/db": "^2.0.0",
     "@web3-storage/multipart-parser": "^1.0.0",
+    "ipfs-car": "^0.5.8",
     "itty-router": "^2.3.10",
     "multiformats": "^9.0.4",
-    "p-retry": "^4.6.1"
+    "p-retry": "^4.6.1",
+    "uint8arrays": "^3.0.0"
   },
   "bundlesize": [
     {
       "path": "./dist/main.js",
-      "maxSize": "1 MB",
-      "compression": "none"
```

Comment:
https://developers.cloudflare.com/workers/platform/limits#script-size

> A Workers script can be up to 1MB in size after compression.

So let's just test this as the limits are defined by Cloudflare

Reply:
i'm getting an error on launching the api in dev mode now about 50% of the time, so we may need to find a lighter s3 client or figure out why cf is less happy

Reply:
it is not related to s3, but to wrangler itself. I will share with you the details